### DISCUSSION TITLE
How to get the YOLO badge ？？？

### DISCUSSION THREAD

**Discussion by chyangyang** [community/community#19530] [Other Feature Feedback, Questions, & Ideas] [answered]
？？

**Comment by YouthguyZ** [comment, marked as answer]
合并未经审查的代码

**Comment by cardison** [comment]
You should install github for dekstop (because this is the easiest way)
    Open any repository on your github desktop and clone it to local storage on your computer
    After that, create a new brach to commit
    Make a change in the new branch, then you commit by including CO-AUTHOR in the commit message column (you can add me to CO-AUTHOR)
    Then do a pull request as usual and merge it. And taaadaa you will get the achievement

**Comment by aaronlamz** [comment]
You just need to:

Open your repository;
Create new branch;
Do some updates;
Then, create pull request to main branch;
IMPORTANT! Add reviewer. To do this, you need to add access to repository someone, then add him as reviewer;
After this, just click merge as you do it always
Check your profile and see yolo acachievement.
More detail here:
https://github.com/github-community/community/discussions/19146

**Reply by Swibee** [reply]
Yeaa
